### ISSUE TITLE
Possibly move vgmstream audacious plugin to audacious-plugin

### ISSUE THREAD

**Issue by kaniini** [losnoco/vgmstream#512]
A thought I had is that it would be possible to move the vgmstream plugin to audacious-plugins.

The main blocker is the lack of a pkg-config file.

What do you all think about it?

**Comment by kode54** [comment]
Sounds like we need that, but I don't know the first thing about making one. Any volunteers?

**Comment by bnnm** [comment]
The advantage of code being here is that basically I'm the only one that does plugin maintenance and I can quickly fix stuff here. Also audacious-plugins seems to use the latest API, while we have users with Audacious 3.6/7 too ([supports both](https://github.com/losnoco/vgmstream/blob/master/audacious/plugin.cc#L227)).

I don't know anything about pkg-config tho.

**Comment by kaniini** [comment]
yes, it is true that audacious-plugins is versioned against specific SDKs (audacious-plugins 4.0.x for example targets the audacious-4.0 SDK).  however, I don't see that as much of a problem.

CMake can generate pkg-config files, but I forget the exact details about how it works.

**Comment by bnnm** [comment]
Well, I guess it'd be a problem for users stuck with an older Audacious, and for us, since they'd come here to ask. Given how niche this project is, I'd rather not break compatibility for the few users unless there is a sound reason.
